integration_tests/import_test.py

```diff
@@ -0,0 +1,118 @@
+import os
+import re
+import subprocess
+
+from keras import backend
+
+BACKEND_REQ = {
+    "tensorflow": "tensorflow",
+    "torch": "torch torchvision",
+    "jax": "jax jaxlib",
+}
+
+
+def setup_package():
+    subprocess.run("rm -rf tmp_build_dir", shell=True)
+    build_process = subprocess.run(
+        "python3 pip_build.py",
+        capture_output=True,
+        text=True,
+        shell=True,
+    )
+    print(build_process.stdout)
+    match = re.search(
+        r"\s[^\s]*\.whl",
+        build_process.stdout,
+    )
+    if not match:
+        raise ValueError("Installing Keras package unsuccessful. ")
+        print(build_process.stderr)
+    whl_path = match.group()
+    return whl_path
+
+
+def create_virtualenv():
+    env_setup = [
+        # Create and activate virtual environment
+        "python3 -m venv test_env",
+        # "source ./test_env/bin/activate",
+    ]
+    os.environ["PATH"] = (
+        "/test_env/bin/" + os.pathsep + os.environ.get("PATH", "")
+    )
+    run_commands_local(env_setup)
+
+
+def manage_venv_installs(whl_path):
+    other_backends = list(set(BACKEND_REQ.keys()) - {backend.backend()})
+    install_setup = [
+        # Installs the backend's package and common requirements
+        "pip install " + BACKEND_REQ[backend.backend()],
+        "pip install -r requirements-common.txt",
+        "pip install pytest",
+        # Ensure other backends are uninstalled
+        "pip uninstall -y "
```

Comment:
If you don't install `requirements` you also don't have to do this, which is a plus

Reply:
This is just an extra check (the virtual environment is fresh, but this ensures there is absolutely no chance of the other backends being installed).